cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for bd")
```

Comment:
There is a typo in the message. It should be 'Waiting for db' instead of 'Waiting for bd'.